### PR TITLE
Bump Upbound Provider AWS to 1.7

### PR DESCRIPTION
**Pull request by artych**
Bump Upbound Provider AWS to 1.7

The upbound_aws_crds.libsonnet list was updated, but no changes in [v1.7.0](https://github.com/crossplane-contrib/provider-upjet-aws/releases/tag/v1.7.0), just resorting.

### CODE REVIEW

File: libs/crossplane/config.jsonnet [hunk 1]
```diff
@@ -97,9 +97,9 @@ config.new(
     // Upbound official providers
     // https://marketplace.upbound.io/
     {
-      output: 'upbound-provider-aws/1.6',
+      output: 'upbound-provider-aws/1.7',
       prefix: '^io\\.upbound\\.aws\\..*',
-      crds: ['https://raw.githubusercontent.com/crossplane-contrib/provider-upjet-aws/v1.6.1/package/crds/%s' % crd for crd in upbound_aws_crds],
+      crds: ['https://raw.githubusercontent.com/crossplane-contrib/provider-upjet-aws/v1.7.0/package/crds/%s' % crd for crd in upbound_aws_crds],
       localName: 'upbound_aws',
```
Comment (jvrplmlmn) [inline libs/crossplane/config.jsonnet:103]: Should we add a comment here somewhere telling the user that if the version is updated, we need to do the same in the Makefile?

Comment (artych) [reply libs/crossplane/config.jsonnet:105]: Added 👍

File: libs/crossplane/upbound_aws_crds.libsonnet [hunk 2]
```diff
@@ -29,8 +29,8 @@
   'apigateway.aws.upbound.io_integrationresponses.yaml',
   'apigateway.aws.upbound.io_integrations.yaml',
   'apigateway.aws.upbound.io_methodresponses.yaml',
-  'apigateway.aws.upbound.io_methodsettings.yaml',
   'apigateway.aws.upbound.io_methods.yaml',
+  'apigateway.aws.upbound.io_methodsettings.yaml',
```
Comment (Duologic) [inline libs/crossplane/upbound_aws_crds.libsonnet:33]: Interesting change, are you on a Mac? Seems like the `sort` command does something different. :)

**Review by Duologic — APPROVED** [review]

File: libs/crossplane/upbound_aws_crds.libsonnet (hunk 2 above)
Comment (artych) [reply libs/crossplane/upbound_aws_crds.libsonnet:33]: Yes I'm on Mac :).

Comment (Duologic) [reply libs/crossplane/upbound_aws_crds.libsonnet:33]: :shrug: It's fine.

**Pull request merged**